pages/assets/[id].tsx

```diff
@@ -0,0 +1,80 @@
+import Head from 'next/head'
+import { useRouter } from 'next/router'
+import { Box, Flex, HStack, NAMED_COLORS, Spinner } from '@ironfish/ui-kit'
+import Breadcrumbs from 'components/Breadcrumbs/Breadcrumbs'
+import { AssetInformationGrid } from 'components/CustomAssets/AssetInformationGrid/AssetInformationGrid'
+import { AssetHistory } from 'components/CustomAssets/AssetHistory/AssetHistory'
+// import { AssetTransactionChart } from 'components/CustomAssets/AssetTransactionChart/AssetTransactionChart'
+import useAsset from 'hooks/useAsset'
+import { useMemo } from 'react'
+import useAssetDescriptions from 'hooks/useAssetDescriptions'
+
+export default function AssetInfo() {
+  const router = useRouter()
+  const { id } = router.query
+
+  const asset = useAsset(id as string)
+
+  const descriptions = useAssetDescriptions({
+    asset: id as string,
+  })
+
+  if (asset.error) {
+    throw asset.error
+  }
+
+  const assetDetails = useMemo(() => {
+    if (!asset.loaded) {
```

Comment:
Can we use same loading state as in block view cards data for asset cards?
https://block-explorer-git-feat-custom-assets-ironfish.vercel.app/blocks/59391